R/tests.R

```diff
@@ -259,6 +259,39 @@ testPlots <- function() {
                    xlab = "PC 1", ylab = "PC 2", main = "PCA")
   }
   assign("plotMultiFeature", plotMultiFeature, envir = parent.frame())
+  plotMultiTest_sf <- function(x) {
+    var_x <- data.frame(lapply(x$results, `[`, 2))
+    colnames(var_x)<- names(x$results)
+    var_x <- data.table::data.table(features = rownames(var_x), var_x)
+    var_x <- data.table::melt(var_x, measure.vars = c(2,3))
+
+    var_y <- data.frame(lapply(x$results, `[`, 3))
+    colnames(var_y) <- names(x$results)
+    var_y <- data.table::data.table(features = rownames(var_y), var_y)
+    var_y <- data.table::melt(var_y, measure.vars = c(2,3))
+
+    df <- merge(var_x, var_y, by=c("variable", "features"))
+
+    plot(df$value.x ~ df$value.y, col = factor(df$variable))
+  }
+  assign("plotMultiTest_sf", plotMultiTest_sf, envir = parent.frame())
+  plotMultiTest_mf <- function(x) {
+    var_x <- data.frame(lapply(x$results, `[`, 2))
+    colnames(var_x)<- names(x$results)
+    var_x <- data.table::data.table(features = rownames(var_x), var_x)
+    var_x <- data.table::melt(var_x, measure.vars = c(2,3))
+
+    var_y <- data.frame(lapply(x$results, `[`, 3))
+    colnames(var_y) <- names(x$results)
+    var_y <- data.table::data.table(features = rownames(var_y), var_y)
+    var_y <- data.table::melt(var_y, measure.vars = c(2,3))
+
+    df <- merge(var_x, var_y, by=c("variable", "features"))
+
+    plot(df$value.x ~ df$value.y, col = factor(df$variable))
```

Comment:
Since this code is inside a package, please add the prefix, ie replace `plot()` with `graphics::plot()`

https://github.com/abbvie-external/OmicNavigator/pull/6/checks?check_run_id=3546104005#step:13:68

Reply:
This is implemented in the newest push